src/test/java/com/github/havardh/javaflow/annotations/JsonProperty.java

```diff
@@ -0,0 +1,11 @@
+package com.github.havardh.javaflow.annotations;
+
+public @interface JsonProperty {
+
+  public final static String USE_DEFAULT_NAME = "";
+
+  String value() default USE_DEFAULT_NAME;
+
+  boolean require() default false;
```

Comment:
This is named `required` in jackson, but I think it's unused here anyways ;)

Reply:
Ok thanks, I'll strip away any unnecessary details :)